src/App.css

```diff
@@ -1,3 +1,367 @@
-.App {
-  color: #000;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300;400;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.ff-m {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.ff-m-b {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 700;
+}
+
+.ff-r-s {
+  font-family: 'Roboto Slab', serif;
+  font-weight: 300;
+}
+
+.ff-r-b {
+  font-family: 'Roboto Slab', serif;
+  font-weight: 700;
+}
+
+.flex {
+  display: flex;
+}
+
+.flex-c {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.header {
+  border-bottom: 0.1rem solid #e8e8e8;
+}
+
+.nav-bar {
+  justify-content: space-between;
+  align-items: center;
+  margin: 1.813rem 6.188rem 1.813rem 6.25rem;
+}
+
+#brand {
```

Comment:
- [ ] [Optional] Try to use class selector as recommended to be one of CSS best practices

Reply:
Done!